.github/CODEOWNERS

```diff
@@ -1 +1 @@
-/pages/ @bethpotts @DanielleSoCompany @peggygannon
+/fragments/home/banner.yml @department-of-veterans-affairs/frontend-review-group
```

Comment:
@department-of-veterans-affairs/frontend-review-group This will most likely never matter for you all, but I'm adding you here as code owners so that if a request to update the homepage banner comes through while I'm out, you all will be notified. 

I say this probably won't matter because soon this will be configurable from the CMS, so it won't require a developer's review.